src/app.ts

```diff
@@ -0,0 +1,36 @@
+import { Application, NextFunction, Request, Response } from 'express';
```

Comment:
u can use import like:
 import express, { Application, NextFunction, Request, Response } from 'express';
and remove redundant require statement